js/solGS/solGS.js

```diff
@@ -566,6 +566,42 @@ solGS.waitPage = function (page, args) {
 	    type    : 'POST',
  	    data    : profile,
 	    url     : '/solgs/run/saved/analysis/',
+	    success : function(response) {
+		if (response.result.match(/Submitted/)) {
+		    confirmRequest();
+		} else {
+		    jQuery('<div />')
+			.html('Error occured submitting the job. Please contact the developers.' + "\n\nHint: " + response.result)
+			.dialog({
```

Comment:
maybe use bootstrap modals instead to standardize how dialogs look in the website.